_topic_maps/_topic_map_osd.yml

```diff
@@ -186,11 +186,32 @@ Name: Networking
 Dir: networking
 Distros: openshift-dedicated
 Topics:
+- Name: Understanding the Ingress Operator
+  File: ingress-operator
+- Name: Configuring the Ingress Controller endpoint publishing strategy
+  File: nw-ingress-controller-endpoint-publishing-strategies
 - Name: OpenShift SDN default CNI network provider
   Dir: openshift_sdn
   Topics:
   - Name: Enabling multicast for a project
     File: enabling-multicast
+- Name: Configuring ingress cluster traffic
+  Dir: configuring_ingress_cluster_traffic
+  Topics:
+  - Name: Overview
```

Comment:
This overview page is describing features like MetalLB and IP failover that is not relevant to OSD/ROSA. I think we should just omit this whole "Configuring ingress cluster traffic" section to not confuse the customer.